src/main/java/gil/server/PostPersonController.java

```diff
@@ -0,0 +1,43 @@
+package gil.server;
+
+import javax.json.JsonObject;
+import java.io.IOException;
+import java.util.function.BiFunction;
+
+public class PostPersonController {
+    public static BiFunction<Request, Response, Response> post =
+            (request, response) -> {
+                JSONDataStore data = new JSONDataStore(new FileIO("people.txt"));
+                String body = request.getBody();
+                JsonObject personJSON = data.getPersonJSONObject(body);
+
+                if (personJSON.containsKey("name") && personJSON.containsKey("email")) {
+                    String name = personJSON.getString("name");
+                    String email = personJSON.getString("email");
+                    Person person = null;
+
+                    try {
+                        person = data.createPerson(name, email);
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+
+                    Integer personId = person.getId();
+                    response.setProtocol("HTTP/1.1");
+                    response.setStatusCode("201");
+                    response.setReasonPhrase("Created");
```

Comment:
I believe you should be able to derive that this is supposed to say "Created" from the status code. You probably dont need to set it each time.

Reply:
I like this idea.